docs/api/security/psa_internal_storage.md

```diff
@@ -0,0 +1,20 @@
+## PSA Internal Storage
+
+### Description
+
+PSA internal storage APIs allows saving and retrieving data from PSA internal flash.
+
+PSA internal storage is implementation varies depending on the target type:
```

Comment:
Who does the implementing?

Reply:
fixed

Reply:
I would add that ITS is intended to be used only by software running in the secure environment.

Reply:
replace the word "implementation" by "functionality"

Reply:
Thanks, @dannybenor !
I've updated the document.